demo/index.html

```diff
@@ -283,10 +297,12 @@ <h5 class="u-text--right">Example</h5>
                             </div>
                         </div>
                         <hr />
-                        <div class="col-8">
-                            <h3>Alignment classes</h3>
-                            <p class="docs-text--success">Easy to re-align blocks of text components with a direction alignment class. Used to place text content in the correct layout or to re-align table data content e.g. Number content is set to right alignment whereas text content is left aligned.</p>
-                            <p class="docs-text--avoid col-8">Avoid using text alignment classes to align blocks of content next to each other. Resort to using block alignment modifiers or the grid system.</p>
+                        <div class="col-12">
+                            <div class="col-8">
```

Comment:
@anthonydillon When I was changing the grid over if I just had an ```col-8``` on it's own it would add the left margin. Encapsulating it in ```col-12``` would remove this as I was tyring to use the new grid like we have in the past.